fabconfig.py

```diff
@@ -1,26 +1,34 @@
-config = dict(    
```

Comment:
you have edited fabfiles used by the upstream developers.  The reason fabdeployfile and fabdeployconfig existed was because otherwise we cannot put in upstream PRs without breaking someone elses stuff